adapters/Base/index.test.js

```diff
@@ -19,33 +21,61 @@ describe('BaseAdapter', () => {
     jest.clearAllMocks();
   });
 
+  describe('authorizationHeader', () => {
+    const mockTokenType = 'token type';
+    const mockAccessToken = 'access token';
+    const mockTokens = {
+      tokenType: mockTokenType,
+      accessToken: mockAccessToken,
+      refreshToken: 'refresh token',
+    };
+
+    beforeEach(() => {
+      getToken.mockImplementation(() => mockTokens);
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('returns the headers that includes the access token', () => {
```

Comment:
Should it be?
```suggestion
    it('returns the header that includes the access token', () => {
```

Reply:
sure, done in fff5bbb3cdc260dcce473d61ad137372948dc0fa! Thanks!